client/src/components/MultiJurisdictionAudit/useSetupMenuItems/index.ts

```diff
@@ -55,8 +55,7 @@ function useSetupMenuItems(
       setContests('processing')
       const condition = async () => {
         const fileProcessing = await getJurisdictionFileStatus(electionId)
-        if (!fileProcessing) return false
-        const { status, error } = fileProcessing
+        const { status, error } = fileProcessing!
```

Comment:
how are you sure this is never null?

Reply:
I might be overassuming, but since we only reach this point in the code if it's already processing the file, the endpoint should never return null, if I understand it right. It only returns null for the processing object on the first call. If something goes wrong after that it returns with an object with an error and so on, but it won't return null.

Reply:
ah i didn't read the context of the rest of the function. yeah I think your reasoning is correct